src/components/tab-title/tab-title.tsx

```diff
@@ -130,6 +175,10 @@ export class TabTitle implements InteractiveComponent {
     }
   }
 
+  componentWillUpdate(): void {
+    this.isFirstLoad = false;
```

Comment:
There might be a better pattern to use here. Would a mutation observer help here?  I'm just wondering why a first load is special in this case?